core/src/main/java/jenkins/security/ResourceDomainRootAction.java

```diff
@@ -327,4 +327,8 @@ private static Token decode(String value) {
     // Not @Restricted because the entire class is
     @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "for script console")
     public static /* not final for Groovy */ int VALID_FOR_MINUTES = SystemProperties.getInteger(ResourceDomainRootAction.class.getName() + ".validForMinutes", 30);
+
+    /* Escape hatch for a fix part of SECURITY-3314 / CVE-2024-23897 (Variant 2) */
```

Comment:
This is misleading, delete.
```suggestion
    /* Escape hatch for a security hardening preventing one of the known ways to elevate arbitrary file read to RCE */
```

Reply:
Draft opened already: https://github.com/jenkins-infra/jenkins.io/pull/7462